src/lib/constants/styles.js

```diff
@@ -19,8 +19,32 @@ export const GEL_SPACING_DBL = '16px';
 // Start defining Reith fonts.
 /* eslint-disable no-unused-expressions */
 injectGlobal`
+  @font-face {
+    font-display: optional;
+    font-family: ReithSansNewsLight;
```

Comment:
Do we need to use this Light Sans? I can't remember seeing it used in the designs.

Reply:
I don't know, but it's only at the cost of the bytes in this file, so why not add it?

Reply:
I agree that currently it won't be that heavy, but if they're available, then we could end up having 8 fonts downloaded on a single article, which would be terrible for performance.

Reply:
I agree, but this will be used by all BBC so this is needed. 8 fonts across all products articles is sensible. I intend to write tests to ensure there is a limit on font-families per route. I will create an issue for this shortly.

Reply:
That sounds good.